tools/dependencies/known-dependencies.txt

```diff
@@ -300,6 +300,7 @@ perfmark-api-0.23.0.jar
 proto-google-common-protos-2.0.1.jar
 protobuf-java-3.17.2.jar
 protobuf-java-util-3.17.2.jar
+protobuf-java-util-3.17.3.jar
```

Comment:
Why do we need to introduce a new version? Since we already have `protobuf-java-util-3.17.2`.